packages/@o3r/styling/builders/style-extractor/schema.ts

```diff
@@ -21,4 +21,7 @@ export interface StyleExtractorBuilderSchema extends SchematicOptionObject {
 
   /** Will not display warning for duplicated variable */
   ignoreDuplicateWarning: boolean;
+
+  /** Will display only a warning for invalid value */
```

Comment:
maybe you should indicate the `@default` as is not really obvious and the `?? true` logic is used in the code